octovis/package.xml

```diff
@@ -19,6 +19,8 @@
   <depend>libqt4-dev</depend>
   <depend>libqt4-opengl-dev</depend>
 
+  <depend>qtbase5-dev</depend>
+
```

Comment:
You need to add `libqt5-core`

Reply:
Do I need to replace it or add it as a new depend?

Reply:
I think add as another, but I haven't tested or looked into it deeply - just checked the correct rosdep key.